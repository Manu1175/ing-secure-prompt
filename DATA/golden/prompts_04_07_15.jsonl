{"id": "prompts_04_07_15:0", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_07_15.xlsx", "modality": "text", "c_level": null, "raw_text": "Create an on-call roster for the payments incident bridge with primary Edward Richardson (+32 37 441 142, edward.richardson@ing.com) and backup Mary Sims (+32 42 361 650, mary.sims@ing.com). Include CorpKeys OE27VJ and NV26ZI.", "expected_scrub": "Create an on-call roster for the payments incident bridge with primary <EMP1_FIRST> <EMP1_LAST> (<EMP1_PHONE>, <EMP1_EMAIL>) and backup <EMP2_FIRST> <EMP2_LAST> (<EMP2_PHONE>, <EMP2_EMAIL>). Include CorpKeys <EMP1_CORPKEY> and <EMP2_CORPKEY>.", "entities": null, "notes": null}
{"id": "prompts_04_07_15:1", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_07_15.xlsx", "modality": "text", "c_level": null, "raw_text": "Build a contact card for ‘CMDB Service Owner’ Mary Sims and ‘Service Desk Lead’ John Graham with phones (+32 42 361 650 / +32 31 603 900) and emails (mary.sims@ing.com / john.graham@ing.com).", "expected_scrub": "Build a contact card for roles with <EMP_A_FIRST> <EMP_A_LAST> and <EMP_B_FIRST> <EMP_B_LAST> including phones (<EMP_A_PHONE> / <EMP_B_PHONE>) and emails (<EMP_A_EMAIL> / <EMP_B_EMAIL>).", "entities": null, "notes": null}
{"id": "prompts_04_07_15:2", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_07_15.xlsx", "modality": "text", "c_level": null, "raw_text": "Draft a request to create distribution list ‘payments-alerts@ing.com’ with initial members: Erin Johnson, Kristy Peterson, Shaun Stanley (include their ing.com emails).", "expected_scrub": "Draft a request to create distribution list '<DL_ALIAS>' with initial members: <EMP_1>, <EMP_2>, <EMP_3> (include their corporate emails).", "entities": null, "notes": null}
{"id": "prompts_04_07_15:3", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_07_15.xlsx", "modality": "text", "c_level": null, "raw_text": "Write an incident paging cascade SMS for Production: notify Daniel Taylor (+32 94 953 438) then Miguel Fitzgerald (+32 81 609 125) if no ack in 10 min.", "expected_scrub": "Write an incident paging cascade SMS for Production: notify <EMP1_NAME> (<EMP1_PHONE>) then <EMP2_NAME> (<EMP2_PHONE>) if no ack in 10 min.", "entities": null, "notes": null}
{"id": "prompts_04_07_15:4", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_07_15.xlsx", "modality": "text", "c_level": null, "raw_text": "Compose a security training reminder email to Heather Johnson (heather.johnson@ing.com) and Nicholas Wolf (nicolas.wolf@ing.com) including CorpKeys NS99DB and LW24YO.", "expected_scrub": "Compose a security training reminder email to <EMP_A_EMAIL> and <EMP_B_EMAIL> including CorpKeys <EMP_A_CORPKEY> and <EMP_B_CORPKEY>.", "entities": null, "notes": null}
{"id": "prompts_04_07_15:5", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_07_15.xlsx", "modality": "text", "c_level": null, "raw_text": "Draft an ops email confirming transfer 7345e701-ff0d-4665-88ef-c45092fcf6e7 Completed (2025-08-14) for EUR 2,215.86 to IBAN BE75 4520 3760 8470.", "expected_scrub": "Draft an ops email confirming transfer <TRANSFER_ID> <STATUS> (<DATE>) for <CURRENCY> <AMOUNT> to IBAN <IBAN>.", "entities": null, "notes": null}
{"id": "prompts_04_07_15:6", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_07_15.xlsx", "modality": "text", "c_level": null, "raw_text": "Send a risk alert for high-value Pending transfer 178b1069-a98a-41d0-896a-5aae18544db4 (EUR 9,672.89) to IBAN BE55 2212 8041 3697.", "expected_scrub": "Send a risk alert for high-value <STATUS> transfer <TRANSFER_ID> (<CURRENCY> <AMOUNT>) to IBAN <IBAN>.", "entities": null, "notes": null}
{"id": "prompts_04_07_15:7", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_07_15.xlsx", "modality": "text", "c_level": null, "raw_text": "Flag IBAN BE62 5474 2787 2767 for review after Failed transfer 93151103-357b-4b8f-8b92-ab7cdab7faaf (EUR 7,813.25) dated 2025-08-16.", "expected_scrub": "Flag IBAN <IBAN> for review after <STATUS> transfer <TRANSFER_ID> (<CURRENCY> <AMOUNT>) dated <DATE>.", "entities": null, "notes": null}
{"id": "prompts_04_07_15:8", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_07_15.xlsx", "modality": "text", "c_level": null, "raw_text": "Prepare a reconciliation note for Pending transfers: be41f293-ae1b-458e-b9d1-5266037b9332 (EUR 9,428.45 to BE14 8365 4959 2937), 717905b1-4d10-4bdb-af7e-3af1cfb9c203 (EUR 5,210.86 to BE15 8901 8689 5763), 4814960b-1640-4885-90db-1c1020d44350 (EUR 8,102.96 to BE60 6027 9010 9265).", "expected_scrub": "Prepare a reconciliation note for <STATUS> transfers: <TRANSFER_ID> (<CURRENCY> <AMOUNT> to <IBAN>), <TRANSFER_ID> (...), <TRANSFER_ID> (...).", "entities": null, "notes": null}
{"id": "prompts_04_07_15:9", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_07_15.xlsx", "modality": "text", "c_level": null, "raw_text": "Provide a compliance sample set: 03488215-4c4a-49aa-ae1c-b7f6ede56a4a (Pending), 7f244a6e-6042-4b1f-b1e4-aced4fc1087d (Completed), 5518158a-0d88-48fb-9ed9-db32800c5216 (Failed), 2fc89765-a638-4b6a-881a-83b5cfa4b877 (Pending), 36d0b951-2d3d-4161-abff-147784fb9e6d (Completed).", "expected_scrub": "Provide a compliance sample set: <TRANSFER_ID> (<STATUS>), <TRANSFER_ID> (<STATUS>), <TRANSFER_ID> (<STATUS>), <TRANSFER_ID> (<STATUS>), <TRANSFER_ID> (<STATUS>).", "entities": null, "notes": null}
{"id": "prompts_04_07_15:10", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_07_15.xlsx", "modality": "text", "c_level": null, "raw_text": "Draft an ops email confirming payment order PO-0038 (Wire Transfer) completed for USD 19,335.18 from Linda Lewis (linda.lewis@hotmail.com) to Supplier 207 at IBAN BE01390286750206 (Ref: Invoice 23294).", "expected_scrub": "Draft an ops email confirming payment order <PAYMENT_ORDER_ID> (<PAYMENT_TYPE>) completed for <CURRENCY> <AMOUNT> from <PAYER_NAME> (<PAYER_EMAIL>) to <PAYEE_NAME> at IBAN <IBAN> (Ref: <REFERENCE>).", "entities": null, "notes": null}
{"id": "prompts_04_07_15:11", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_07_15.xlsx", "modality": "text", "c_level": null, "raw_text": "Open a failure investigation for PO-0031 (Wire Transfer GBP 1,382.38) to Supplier 558 (IBAN BE27905874889097), Status: Failed. Ask for reason codes and retry window.", "expected_scrub": "Open a failure investigation for <PAYMENT_ORDER_ID> (<PAYMENT_TYPE> <CURRENCY> <AMOUNT>) to <PAYEE_NAME> (IBAN <IBAN>), Status: <STATUS>. Ask for reason codes and retry window.", "entities": null, "notes": null}
{"id": "prompts_04_07_15:12", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_07_15.xlsx", "modality": "text", "c_level": null, "raw_text": "Prepare a Pending Standing Order digest: PO-0044 EUR 6,242.15 (Linda Smith), PO-0029 USD 19,031.16 (Tom Jones), PO-0041 EUR 14,825.92 (Linda Jones).", "expected_scrub": "Prepare a <STATUS> <PAYMENT_TYPE> digest: <PAYMENT_ORDER_ID> <CURRENCY> <AMOUNT> (<PAYEE_OR_PAYER>), ...", "entities": null, "notes": null}
{"id": "prompts_04_07_15:13", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_07_15.xlsx", "modality": "text", "c_level": null, "raw_text": "Draft an analysis of Direct Debit failures: PO-0047 (GBP 1,531.94 to Supplier 186, IBAN BE35825771477528) and PO-0049 (GBP 18,071.87 to Supplier 207, IBAN BE59083717722036).", "expected_scrub": "Draft an analysis of <PAYMENT_TYPE> failures: <PAYMENT_ORDER_ID> (<CURRENCY> <AMOUNT> to <PAYEE_NAME>, IBAN <IBAN>) and <PAYMENT_ORDER_ID> (<CURRENCY> <AMOUNT> to <PAYEE_NAME>, IBAN <IBAN>).", "entities": null, "notes": null}
{"id": "prompts_04_07_15:14", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_07_15.xlsx", "modality": "text", "c_level": null, "raw_text": "Create a Cheque completion digest: PO-0013 GBP 13,990.49; PO-0014 USD 1,632.00; PO-0015 USD 12,595.45; PO-0034 EUR 5,835.07; PO-0054 USD 2,792.05.", "expected_scrub": "Create a <PAYMENT_TYPE> completion digest: <PAYMENT_ORDER_ID> <CURRENCY> <AMOUNT>; ...", "entities": null, "notes": null}
{"id": "prompts_04_07_15:15", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_07_15.xlsx", "modality": "text", "c_level": null, "raw_text": "Draft a payer notification for PO-0040 (Direct Debit Completed, USD 2,090.09) initiated by Tom Clark to Supplier 776 (IBAN BE96081490396036).", "expected_scrub": "Draft a payer notification for <PAYMENT_ORDER_ID> (<PAYMENT_TYPE> <STATUS>, <CURRENCY> <AMOUNT>) initiated by <PAYER_NAME> to <PAYEE_NAME> (IBAN <IBAN>).", "entities": null, "notes": null}
{"id": "prompts_04_07_15:16", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_07_15.xlsx", "modality": "text", "c_level": null, "raw_text": "Create a risk note for higher amounts: PO-0001 EUR 15,660.40 (Completed, Wire), PO-0011 EUR 19,686.97 (Failed, Standing Order), PO-0038 USD 19,335.18 (Completed, Wire).", "expected_scrub": "Create a risk note for higher amounts: <PAYMENT_ORDER_ID> <CURRENCY> <AMOUNT> (<STATUS>, <PAYMENT_TYPE>), ...", "entities": null, "notes": null}
{"id": "prompts_04_07_15:17", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_07_15.xlsx", "modality": "text", "c_level": null, "raw_text": "Notify Asset Owner Mary Sims (mary.sims@ing.com) that payment order PO-0033 failed; ask to coordinate with Treasury Ops.", "expected_scrub": "Notify Asset Owner <ASSET_OWNER> (<ASSET_OWNER_EMAIL>) that payment order <PAYMENT_ORDER_ID> failed; ask to coordinate with Treasury Ops.", "entities": null, "notes": null}
{"id": "prompts_04_07_15:18", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_07_15.xlsx", "modality": "text", "c_level": null, "raw_text": "Create a data-minimization note: IBANs appearing in transfers or payment orders (e.g., BE75 4520 3760 8470; BE01390286750206) must be masked (show last 4) in analytics exports and restricted to least-privilege roles.", "expected_scrub": "Create a data-minimization note: IBANs appearing in transfers or payment orders (e.g., <IBAN_EX_1>; <IBAN_EX_2>) must be masked (show last 4) in analytics exports and restricted to least-privilege roles.", "entities": null, "notes": null}
{"id": "prompts_04_07_15:19", "source_path": "external/SecurePrompt/PROMPTS/prompts_04_07_15.xlsx", "modality": "text", "c_level": null, "raw_text": "Create a data-minimization note: when exporting employee phone numbers/emails (e.g., mary.sims@ing.com, +32 42 361 650), use pseudonymous IDs in analytics and restrict direct-contact fields to HR roles.", "expected_scrub": "Create a data-minimization note: when exporting employee phone numbers/emails (e.g., <EMP_EMAIL>, <EMP_PHONE>), use pseudonymous IDs in analytics and restrict direct-contact fields to HR roles.", "entities": null, "notes": null}
