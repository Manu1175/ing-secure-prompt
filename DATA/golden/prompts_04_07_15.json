[
  {
    "text": "Create an on-call roster for the payments incident bridge with primary Edward Richardson (+32 37 441 142, edward.richardson@ing.com) and backup Mary Sims (+32 42 361 650, mary.sims@ing.com). Include CorpKeys OE27VJ and NV26ZI.",
    "labels": [
      "EMAIL",
      "PHONE"
    ]
  },
  {
    "text": "Build a contact card for ‘CMDB Service Owner’ Mary Sims and ‘Service Desk Lead’ John Graham with phones (+32 42 361 650 / +32 31 603 900) and emails (mary.sims@ing.com / john.graham@ing.com).",
    "labels": [
      "EMAIL",
      "PHONE"
    ]
  },
  {
    "text": "Draft a request to create distribution list ‘payments-alerts@ing.com’ with initial members: Erin Johnson, Kristy Peterson, Shaun Stanley (include their ing.com emails).",
    "labels": [
      "EMAIL"
    ]
  },
  {
    "text": "Write an incident paging cascade SMS for Production: notify Daniel Taylor (+32 94 953 438) then Miguel Fitzgerald (+32 81 609 125) if no ack in 10 min.",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Compose a security training reminder email to Heather Johnson (heather.johnson@ing.com) and Nicholas Wolf (nicolas.wolf@ing.com) including CorpKeys NS99DB and LW24YO.",
    "labels": [
      "EMAIL",
      "PHONE"
    ]
  },
  {
    "text": "Draft an ops email confirming transfer 7345e701-ff0d-4665-88ef-c45092fcf6e7 Completed (2025-08-14) for EUR 2,215.86 to IBAN BE75 4520 3760 8470.",
    "labels": [
      "DOB",
      "PHONE"
    ]
  },
  {
    "text": "Send a risk alert for high-value Pending transfer 178b1069-a98a-41d0-896a-5aae18544db4 (EUR 9,672.89) to IBAN BE55 2212 8041 3697.",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Flag IBAN BE62 5474 2787 2767 for review after Failed transfer 93151103-357b-4b8f-8b92-ab7cdab7faaf (EUR 7,813.25) dated 2025-08-16.",
    "labels": [
      "DOB",
      "PHONE"
    ]
  },
  {
    "text": "Prepare a reconciliation note for Pending transfers: be41f293-ae1b-458e-b9d1-5266037b9332 (EUR 9,428.45 to BE14 8365 4959 2937), 717905b1-4d10-4bdb-af7e-3af1cfb9c203 (EUR 5,210.86 to BE15 8901 8689 5763), 4814960b-1640-4885-90db-1c1020d44350 (EUR 8,102.96 to BE60 6027 9010 9265).",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Provide a compliance sample set: 03488215-4c4a-49aa-ae1c-b7f6ede56a4a (Pending), 7f244a6e-6042-4b1f-b1e4-aced4fc1087d (Completed), 5518158a-0d88-48fb-9ed9-db32800c5216 (Failed), 2fc89765-a638-4b6a-881a-83b5cfa4b877 (Pending), 36d0b951-2d3d-4161-abff-147784fb9e6d (Completed).",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Draft an ops email confirming payment order PO-0038 (Wire Transfer) completed for USD 19,335.18 from Linda Lewis (linda.lewis@hotmail.com) to Supplier 207 at IBAN BE01390286750206 (Ref: Invoice 23294).",
    "labels": [
      "EMAIL",
      "PHONE"
    ]
  },
  {
    "text": "Open a failure investigation for PO-0031 (Wire Transfer GBP 1,382.38) to Supplier 558 (IBAN BE27905874889097), Status: Failed. Ask for reason codes and retry window.",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Prepare a Pending Standing Order digest: PO-0044 EUR 6,242.15 (Linda Smith), PO-0029 USD 19,031.16 (Tom Jones), PO-0041 EUR 14,825.92 (Linda Jones).",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Draft an analysis of Direct Debit failures: PO-0047 (GBP 1,531.94 to Supplier 186, IBAN BE35825771477528) and PO-0049 (GBP 18,071.87 to Supplier 207, IBAN BE59083717722036).",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Create a Cheque completion digest: PO-0013 GBP 13,990.49; PO-0014 USD 1,632.00; PO-0015 USD 12,595.45; PO-0034 EUR 5,835.07; PO-0054 USD 2,792.05.",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Draft a payer notification for PO-0040 (Direct Debit Completed, USD 2,090.09) initiated by Tom Clark to Supplier 776 (IBAN BE96081490396036).",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Create a risk note for higher amounts: PO-0001 EUR 15,660.40 (Completed, Wire), PO-0011 EUR 19,686.97 (Failed, Standing Order), PO-0038 USD 19,335.18 (Completed, Wire).",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Notify Asset Owner Mary Sims (mary.sims@ing.com) that payment order PO-0033 failed; ask to coordinate with Treasury Ops.",
    "labels": [
      "EMAIL",
      "PHONE"
    ]
  },
  {
    "text": "Create a data-minimization note: IBANs appearing in transfers or payment orders (e.g., BE75 4520 3760 8470; BE01390286750206) must be masked (show last 4) in analytics exports and restricted to least-privilege roles.",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Create a data-minimization note: when exporting employee phone numbers/emails (e.g., mary.sims@ing.com, +32 42 361 650), use pseudonymous IDs in analytics and restrict direct-contact fields to HR roles.",
    "labels": [
      "EMAIL",
      "PHONE"
    ]
  }
]